docs/jsons.rst

```diff
@@ -316,6 +301,7 @@ This fields are responsible for describing types/scales of plot X and Y axis
 
 * **defaultYAxisType** (*string*, ["log", "linear"]) - specifies scale of Y-axis, if not provided "linear" will be applied
 
+* **plotType** (*string*, ["lines", "points"]) - radio button that allows to determine if plot will be display as points or line. When not provided button will not be displayed and plot type will be "line"
```

Comment:
Yes, it may be confusing because I used names from ant.design documentation for it - [link](https://ant.design/components/radio/).

This element looks like button but can be select only one like in your example:
![image](https://user-images.githubusercontent.com/23633694/50540548-224c4580-0b94-11e9-9d98-df9b56f1e360.png)

Should I change that name?